drv/opentitan-aontimer/src/lib.rs

```diff
@@ -0,0 +1,150 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#![no_std]
+
+#[allow(unused_imports)]
+use userlib::*;
+// use zerocopy::AsBytes;
+
+use core::ptr::slice_from_raw_parts_mut;
+
+const WDOG_REGWEN: usize = 4;
```

Comment:
These should all be `u32`. They are fixed width 32-bit as per specification.

Reply:
Fixed.

Reply:
Actually, these are just indexes into an array, not the registers themselves.  With adjust the name to reflect this, as it also confused me.

Reply:
Ack